ncm-spma/src/main/perl/spma/yumng.pm

```diff
@@ -428,19 +428,21 @@ sub Configure
     # Continue only if package content is supposed to be changed
     return 1 unless $t->{run};
 
+    my $yum_test_chroot = tempdir("spma_yum_testroot_XXXXXX", TMPDIR => 1, CLEANUP => 1);
+    my $yum_tmpdir = tempdir("spma_yum_tmpdir_XXXXXX", TMPDIR => 1, CLEANUP => 1);
+    local $ENV{TMPDIR} = $yum_tmpdir;
+
     # Run test transaction to get complete list of packages to be present on the system
-    $self->execute_command(["rm -rf " . YUM_TEST_CHROOT], "cleaning YUM test chroot", 1);
-    $self->execute_command(["mkdir -p " . YUM_TEST_CHROOT . "/var/cache"],                 "setting up YUM test chroot",    1);
-    $self->execute_command(["ln -s /var/cache/yum " . YUM_TEST_CHROOT . "/var/cache/yum"], "setting YUM test chroot cache", 1);
-    my $yum_install_test_command = "yum install " . YUM_PLUGIN_OPTS . " -C --installroot=" . YUM_TEST_CHROOT;
+    $self->execute_command(["mkdir -p " . $yum_test_chroot . "/var/cache"],                 "setting up YUM test chroot",    1);
```

Comment:
Now these are no longer constants, I think interpolation would be a lot easier to read than concatenation here?

Reply:
Agreed, next time :-)